Documentation/architecture.md

```diff
@@ -50,18 +50,28 @@ A UnitState object represents the state of a Unit in the fleet engine. A UnitSta
 
 ## Preview Release
 
-Current releases of fleet don't currently perform any authentication or authorization for submitted units. This means that any client that can access your etcd cluster can potentially run arbitrary code on many of your machines very easily.
+Current releases of fleet don't currently perform any authentication or authorization for submitted units. This means that any client that can access your etcd cluster can potentially run arbitrary code on many of your machines very easily, thus it is strongly recommended to enable [TLS authentication][fleet-tls] on the etcd side and set proper file permissions to the keypair on the host.
 
 ## Securing etcd
 
 You should avoid public access to etcd and instead run fleet [from your local laptop][using-the-client] with the `--tunnel` flag to run commands over an SSH tunnel. You can alias this flag for easier usage: `alias fleetctl=fleetctl --tunnel 10.10.10.10` - or use the environment variable `FLEETCTL_TUNNEL`.
 
-## Other Notes
+## Securing fleet
 
-Since it interacts directly with systemd over D-Bus, the fleetd daemon must be run with elevated privileges (i.e. as root) in order to perform operations like starting and stopping services. From the [systemd D-Bus documentation][systemd-dbus]:
+It is also recommended to run fleetd under separate `fleet` user and group, and set listening unix socket permissions to `0660`. This will require local user to be in `fleet` group to perform an action with fleetd. While fleet daemon uses [D-Bus][d-bus] it is necessary to create a [`polkit(8)`][polkit] rule to allow fleetd to communicate with systemd:
```

Comment:
s/While fleet daemon/Since the fleet daemon uses D-Bus to communicate with systemd/